modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -120,10 +128,18 @@
 
 	subtler_message = span_emote("<b>[user]</b>[space]<i>[user.say_emphasis(subtler_message)]</i>")
 
-	if(emote_type == EMOTE_AUDIBLE)
-		user.audible_message_subtler(message = subtler_message, hearing_distance = 1, ignored_mobs = GLOB.dead_mob_list, separation = space)
+	if(istype(target))
+		user.show_message(subtler_message, alt_msg = subtler_message)
+		if(get_dist(user.loc, target.loc) <= SUBTLE_DEFAULT_DISTANCE)
+			target.show_message(subtler_message, alt_msg = subtler_message)
+		else
+			to_chat(user, "Your emote was unable to be sent to your target: Too far away.", type = MESSAGE_TYPE_WARNING)
 	else
-		user.visible_message(message = subtler_message, self_message = subtler_message, vision_distance = 1, ignored_mobs = GLOB.dead_mob_list, separation = space)
+		var/ghostless = get_hearers_in_view(target, user) - GLOB.dead_mob_list
+		for(var/mob/reciever in ghostless)
+			reciever.show_message(subtler_message, alt_msg = subtler_message)
+
+	return TRUE
```

Comment:
What's the purpose of returning `TRUE` here?

Reply:
It already returned true before, and I wasn't about to dive into emote code at the time to figure out why.

Reply:
Yea, emote code returns true on a successful emote.